src/main/java/com/mentorship/vineservice/dto/IdDto.java

```diff
@@ -0,0 +1,14 @@
+package com.mentorship.vineservice.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
+public class IdDto {
```

Comment:
I think this dto is redundant. We can just return Long without wrapping into a separate class